images/runtime/download-cni.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+cni_version="0.7.5"
```

Comment:
Might be nice to factor all of these version defines into a separate shell script purely for versions and import that into each of these shell scripts. That way we have exactly one place to check for all of our versions and we can iterate through & evaluate whether to bump them.

Reply:
(I see at least ineffassign, gops and cni here so far, I'm guessing there'll be more)

Reply:
Yeah, definitely something to think about, for now I just make sure versions are set as variables at the top (and `ARGS` in Dockerfile).

The current take overall is that each of the images that are self-contained to a single directory is defined by that directory, hence the use of tree hashes for image versioning.

I am actually thinking it'd be nice to have a YAML file with versions and other build prams set out, and allow developers to override it locally for testing.